src/DynamoManipulation/NodeManipulator.cs

```diff
@@ -134,7 +134,7 @@ internal IRenderPackageFactory RenderPackageFactory
         /// <param name="gizmoInAction">The Gizmo in action.</param>
         /// <param name="offset">Offset amount by which Gizmo has moved.</param>
         /// <returns>New expected position of the Gizmo</returns>
-        protected abstract Point OnGizmoMoved(IGizmo gizmoInAction, Vector offset);
```

Comment:
We can still make this method return the new point. Earlier this method was not supposed to change the state of the manipulator, however now it will change the state.